test/integration-runner.env

```diff
@@ -39,7 +39,7 @@ SIMULATOR_REMOTE_PORT=${SIMULATOR_REMOTE_PORT:-"8444"}
 CENTRAL_LEDGER_HOST=${CENTRAL_LEDGER_HOST:-"central-ledger-int"}
 CENTRAL_LEDGER_PORT=${CENTRAL_LEDGER_PORT:-"3001"}
 CENTRAL_LEDGER_IMAGE=${CENTRAL_LEDGER_IMAGE:-'mojaloop/central-ledger'}
-CENTRAL_LEDGER_TAG=${CENTRAL_LEDGER_TAG:-'latest'}
+CENTRAL_LEDGER_TAG=${CENTRAL_LEDGER_TAG:-'v9.1.1-snapshot'}
```

Comment:
I suggest waiting for the central-ledger merge and release and changing this back to latest